python/lsst/pipe/base/executionButlerBuilder.py

```diff
@@ -150,7 +152,7 @@ def _accumulate(
         if dataset_type.component() is not None:
             dataset_type = dataset_type.makeCompositeDatasetType()
         dataset_type = _validate_dataset_type(dataset_type, datasetTypes, butler.registry)
-        inserts[dataset_type].add(DataCoordinate.makeEmpty(dataset_type.dimensions.universe))
+        init_inserts[dataset_type].add(DataCoordinate.makeEmpty(dataset_type.dimensions.universe))
```

Comment:
I'm a bit confused about the initOutputs. Do they have resolved refs in the graph? When I ran tests for this in `pipelines_check` it seemed that `icSrc_schema` was in the outputs as well and so was picked up below explicitly but `packages` and `isr_config` was only here.

Reply:
Yes, `icSrc_schema` is an `initOutput` of `characterizeImage`, and an `initInput` of `calibrate`>

Reply:
Okay. So I'm assuming `packages` and `*_config` must be in the graph. I think this part is only working because we are still unresolving on put.